DistFiles/localizations/Chorus.en.tmx

```diff
@@ -475,7 +468,7 @@
     </tu>
     <tu tuid="Messages.ChorusServerIncompatible">
       <tuv xml:lang="en">
-        <seg>The Chorus Server is not compatible with this client.</seg>
+        <seg>The Chorus Server is not compatible with ths client.</seg>
```

Comment:
I "reqest" you to fix "ths" and the other spelling errors the generator "encounterd"